components/console-backend-service/internal/domain/apigateway/extractor/apirule.go

```diff
@@ -0,0 +1,49 @@
+package extractor
```

Comment:
If possible follow the convention from functions (possibly from PR-6082: https://github.com/kyma-project/kyma/pull/6082/files#diff-6d6718dd1a82f059a07887edc1561eb6R1)
This includes for example:
- renaming file from extractor/apirule.go to apirule/convert.go
- renaming package from extractor to apirule
- renaming package in tests from extractor_test to apirule_test
- making methods private (check how it's used in functions https://github.com/kyma-project/kyma/pull/6082/files#diff-091a86ca44557984c6860916fb76d4f3R43)
- is Do method necessary? 
- probably methods from converter can be moved here

Reply:
done